repairnator/src/main/java/fr/inria/spirals/repairnator/serializer/gsheet/GoogleSpreadSheetFactory.java

```diff
@@ -33,8 +33,9 @@
     private Logger logger = LoggerFactory.getLogger(GoogleSpreadSheetFactory.class);
     private static final String APPLICATION_NAME = "RepairNator Bot";
     private static final File DATA_STORE_DIR = new File(System.getProperty("user.home"), ".credentials/sheets.googleapis.com-repairnator");
-    private static final String GOOGLE_SECRET_PATH = "./client_secret.json";
-    public static final String SPREADSHEET_ID = "1FUHOVx1Y3QZCAQpwcrnMzbpmMoWTUdNg0KBM3NVL_zA";
+    private static final String GOOGLE_SECRET_PATH = "/home/fernanda/git/librepair/repairnator/client_secret.json";
+    //public static final String SPREADSHEET_ID = "1FUHOVx1Y3QZCAQpwcrnMzbpmMoWTUdNg0KBM3NVL_zA";
+    public static final String SPREADSHEET_ID = "1MnRwoZGCxxbmkiswc0O6Rg43wJFTBc3bIyrNdTiBhQ4";
 
```

Comment:
`GOOGLE_SECRET_PATH` is a path for a file, so it depends on your environment: we can define a default value but the real value should be pass as an argument when we launch the program. You should never have an absolute path in the code.

Concerning the `SPREADSHEET_ID` it's an open question: maybe we could use constants and get the right one depending on the mode, as you propose. Maybe you will create a specific class for the serializer and then each serializer has its own spreadsheet_id value.